### PR TITLE
[IMP] theme_real_estate: theme improvement

### PR DESCRIPTION
**Pull request by anso-odoo**
Revamp of `theme_real_estate`

task-4178611

| Before | After |
|--------|--------|
| real_estate_screenshot 2 | real_estate_screenshot |

### CODE REVIEW

File: theme_real_estate/views/snippets/s_title.xml [hunk 1]
```diff
@@ -4,7 +4,16 @@
 <template id="s_title" inherit_id="website.s_title">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="pt72" remove="pt40" separator=" "/>
+        <attribute name="class" add="pt120 pb120 o_cc o_cc4" remove="pt40 pb40" separator=" "/>
+        <attribute name="data-oe-shape-data">{'shape':'web_editor/Airy/07','showOnMobile':true}</attribute>
+    </xpath>
+    <!-- Shape -->
+    <xpath expr="//div[hasclass('container')]" position="before">
+        <div class="o_we_shape o_web_editor_Airy_07 o_we_animated" style="background-image: url('/web_editor/shape/web_editor/Airy/07.svg"/>
```
Comment (xO-Tx) [inline theme_real_estate/views/snippets/s_title.xml:12]: I don't think you need the `o_we_animated`  class for this shape...

**Review by xO-Tx — CHANGES_REQUESTED** [review]
@anso-odoo some adjustments needed here and it'll be ready to go :+1:

**Comment by anso-odoo** [comment]
Thanks @xO-Tx, fixed everything 👍

**Comment by qsm-odoo** [comment]
@robodoo delegate=xO-Tx 
@xO-Tx

**Comment by xO-Tx** [comment]
@robodoo r+

**Comment by xO-Tx** [comment]
@robodoo r-

**Comment by xO-Tx** [comment]
@robodoo r+

**Pull request closed without merging**